tests/core/test_kontainer_drivers.py

```diff
@@ -94,9 +95,73 @@ def test_enabling_driver_exposes_schema(admin_mc, remove_resource):
     kd.active = False
     admin_mc.client.update_by_id_kontainerDriver(kd.id, kd)
 
+    admin_mc.client.delete(kd)
+    verify_driver_not_in_types(admin_mc.client, kd)
```

Comment:
Fix remove resource so it properly removes the kd rather than this explicit removal here.